atomisticparsers/utils/parsers.py

```diff
@@ -200,3 +201,41 @@ def parse_md_workflow(self, data: Dict[str, Any]) -> None:
         sec_workflow = MolecularDynamics()
         self.parse_section(data, sec_workflow)
         self.archive.workflow2 = sec_workflow
+
+    def parse_interactions(self, interactions: Dict[str, Any], sec_model: MSection) -> None:
+
+        interaction_key_list = Interaction.__dict__.keys()
+        interaction_dict = {}
+        interaction_keys_remove = ['__module__', '__doc__', 'm_def']
+        interaction_key_list = [key for key in interaction_key_list if key not in interaction_keys_remove]
```

Comment:
you can use the inspect module here, inspect.getmembers(Interaction). But I suggest you simply list down all the class attributes since from line 222-232, you also have key-specific actions.

Reply:
I thought it would be nicer to get these automatically. The key-specific actions are only a small subset which have special data structures. But I don't see any easy way to only grab the keys that I want with inspect, and I also agree that this current way is not very nice. So, I just made a hard-coded list as suggested. If you have any other ideas let me know, otherwise I will leave it like this.

Reply:
normally you do not access __dict__.  you can do this so you do not have to remove keys.
```python
interaction_keys = [n for n, q in inspect.getmembers(Interaction) if isinstance(q, Quantity)]

Reply:
or if you do not need the keys, simply do
``python
if hasattr(sec_interaction, key):
    sec_interaction.m_set(sec_interaction.m_get_quantity_definition(key), val)

Reply:
Ok, thank you. I have went ahead and used your suggestion with `inspect` now...done.